### PR TITLE
Palindromic_string_added 2 C/C++

### LINKED ISSUE #1706: code to check if a string is palindrome or not in C and C++ .
I can do this.
Assign me this issue. :)

### PR DESCRIPTION
**Pull request by MdAkdas**
Fixes #1706

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- C
- C++

#### Files Added:
- palindromic_string.c 
- palindromic_string.cpp 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: palindromic_string/palindromic_string.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    
+    length = strlen(string1);
+    
+    for(i = 0;i < length/2 ;i++)
+    {
+        if(string1[i] != string1[length-i-1])
+        {
+            flag = 1;
+            break;
+  	}
```
Comment (raghav-dalmia) [inline palindromic_string/palindromic_string.cpp]: indentation issue

File: palindromic_string/palindromic_string.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    char string1[20];
+    int i, length;
+    int flag = 0;
+    
+    cout << "Enter a string: ";
+    cin >> string1;
+    
+    length = strlen(string1);
+    
+    for(i = 0;i < length/2 ;i++)
```
Comment (raghav-dalmia) [inline palindromic_string/palindromic_string.cpp]: add space around ;

File: palindromic_string/palindromic_string.cpp [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+#include <iostream>
+#include<cstring>
+using namespace std;
+
+// Driver function
+int main()
+{
+    char string1[20];
+    int i, length;
+    int flag = 0;
```
Comment (raghav-dalmia) [inline palindromic_string/palindromic_string.cpp]: use bool datatype for flag.

File: palindromic_string/palindromic_string.c [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    char string1[20];
+    int i, length;
+    int flag = 0;
+    
+    printf("Enter a string:");
+    scanf("%s", string1);
+    
+    length = strlen(string1);
+    
+    for(i=0;i < length/2 ;i++)
```
Comment (raghav-dalmia) [inline palindromic_string/palindromic_string.c]: add space around ;

**Comment by raghav-dalmia** [comment]
add issue number in PR description. Also, squash your commits.

**Comment by MdAkdas** [comment]
@raghav-dalmia @asha15 @jainaman224  please review.

File: palindromic_string/palindromic_string.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    char string1[20];
+    int i, length;
+    bool flag = false;
+    
+    printf("Enter a string: ");
+    scanf("%s", string1);
+    
+    length = strlen(string1);
+    
+    for(i=0 ; i < length/2 ; i++)
```
Comment (asha15) [inline palindromic_string/palindromic_string.c]: remove the space before ;
keep space around operators

File: palindromic_string/palindromic_string.c [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        if(string1[i] != string1[length-i-1])
```
Comment (asha15) [inline palindromic_string/palindromic_string.c]: keep space around operators

File: palindromic_string/palindromic_string.cpp [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    bool flag = false;
+    
+    cout << "Enter a string: ";
+    cin >> string1;
+    
+    length = strlen(string1);
+    
+    for(i = 0 ; i < length/2 ; i++)
+    {
+        if(string1[i] != string1[length-i-1])
```
Comment (asha15) [inline palindromic_string/palindromic_string.cpp]: remove the space before ;
keep space around operators

**Comment by MdAkdas** [comment]
@asha15, @raghav-dalmia  please review this again.

**Pull request merged**